tools/python_tools/cuvslam_tools/dataset_preparation/rgbd.py

```
@@ -0,0 +1,484 @@
+The SE(3) and quaternion helpers duplicate private equivalents in
+``euroc/convert_euroc.py``. Folding EuRoC onto these is a separate cleanup;
+changing the EuRoC converter here would put an unrelated dataset's output at
+risk.
+"""
+
+import bisect
+import json
+import math
+from dataclasses import dataclass
+from decimal import Decimal, InvalidOperation
+from typing import Dict, List, Optional, Sequence, Tuple
```

Ruff (0.16.2)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)